tests/testthat/setup.R

```diff
@@ -14,8 +14,14 @@ sim_obs <- simulate_gillespie() |>
     meanlog = meanlog,
     sdlog = sdlog
   ) |>
-  observe_process() |>
-  dplyr::filter(.data$stime_upr <= obs_time) |>
+  dplyr::mutate(
+    ptime_lwr = floor(.data$ptime),
+    ptime_upr = .data$ptime_lwr + 1,
+    stime_lwr = floor(.data$stime),
+    stime_upr = .data$stime_lwr + 1,
+    obs_time = obs_time
```

Comment:
why add obs_time vs using as a variable as previously?

Reply:
I think later on in the code it is used as a part of the dataframe (this is what `observe_process` was doing, so previously it was not used as a variable I think)

Reply:
its coded as a variable on  line 3 - if you just take it out of the mutate call I think it should all just work

Reply:
In tests perhaps `obs_time` is needed in the `data.frame` (not only for the filtering)